content/post/stash-v2023.02.28 /index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Introducing Stash v2023.02.28
+date: "2023-02-28"
+weight: 10
+authors:
+- Hossain Mahmud
+tags:
+- backup
+- kubernetes
+- restore
+- stash
+---
+
+We are announcing Stash `v2023.02.28` which includes a few bug fixes. You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2023.02.28/README.md). In this post, we are going to highlight the changes.
+
+### Bug Fixes
+
+- Fixed a bug causing Batch Backups to get stuck in the Running phase when using the `VolumeSnapshotter` driver ([#1500](https://github.com/stashed/stash/pull/1500)).
```

Comment:
Change `Batch Backups` to `batch backup`. Also, either bold/inline code `Running` phse.